docs/use.rst

```diff
@@ -247,6 +247,7 @@ Other settings
 By default if no tenant is found it will raise an error Http404 however you add ```SHOW_PUBLIC_IF_NO_TENANT_FOUND``` to
 the setting it will display the the public tenant. This will not work for subfolders.
 
+```DEFAULT_NOT_FOUND_TENANT_VIEW``` If set, specifies a path to a view (function-based or class-based) that will handle requests when no tenant is found for the current domain. `DEFAULT_NOT_FOUND_TENANT_VIEW='myapp.views.my_view'`
```

Comment:
Given the information provided I think it is worth it to explictly document a warning that whetever this custom view is it will be executed in the public schema so that anyone wanting to use it is aware of it!